lms/djangoapps/pocs/models.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from student.models import CourseEnrollment, AlreadyEnrolledError
+from xmodule_django.models import CourseKeyField, LocationKeyField
+
+
+class PersonalOnlineCourse(models.Model):
+    """
+    A Personal Online Course.
+    """
+    course_id = CourseKeyField(max_length=255, db_index=True)
+    display_name = models.CharField(max_length=255)
+    coach = models.ForeignKey(User, db_index=True)
+
+
+class PocMembership(models.Model):
+    """
+    Which students are in a POC?
+    """
+    poc = models.ForeignKey(PersonalOnlineCourse, db_index=True)
+    student = models.ForeignKey(User, db_index=True)
```

Comment:
An override contains reference to the specific POC for which it has been created and the field and location to which it belongs.  So for a given [POC, field and location](https://github.com/edx/edx-platform/pull/6636/files#diff-1d8c2a4e9d44739ca2967dc6dc804ad4R69), there will be only one override.  If a students belongs to multiple POCs, when they are viewing a given poc, they will get the overrides that [apply to that POC](https://github.com/edx/edx-platform/pull/6636/files#diff-23df09fae1731c41f5a8e8e697c594b1R23), not ones that are from another POC.

Does that answer the question? If not, what am I missing?